README.md

```diff
@@ -1052,18 +1052,18 @@ Other Style Guides
 
     ```javascript
     // bad
-    const itemHeight = item => item.height > 256 ? item.largeSize : item.smallSize;
+    const itemHeight = item => item.height <= 256 ? item.largeSize : item.smallSize;
```

Comment:
I don’t agree; i think the reverse of the operator shows it better than half of the tokens of the operator. Do you have more rationale for this change than that?